MergerService/Utils/TaskUtils.cs

```diff
@@ -53,18 +52,16 @@ public class TaskUtils : ITaskUtils
                 string url = new Uri(new Uri(_jobManagerUrl), relativeUri).ToString();
                 string? taskData = this._httpClient.PostData(url, null);
 
-                if (taskData is null)
-                {
-                    return null;
-                }
```

Comment:
Why was this moved from here?

Reply:
combined to single line, but i will revert it so it could be more clear.